### DISCUSSION TITLE
403 Error

### DISCUSSION THREAD

**Discussion by cnewfeldt** [gotenberg/gotenberg#858] [Q&A] [answered]
installed the docker instance to my Unraid box and using the CURL script example i'm being returned this error. 

{"level":"error","ts":1713554534.325497,"logger":"api","msg":"convert URL to PDF: convert to PDF: filter URL: 'https://sparksuite.github.io/simple-html-invoice-template/' does not match the expression from the allowed list: value filtered","trace":"d2853b29-d54e-4342-bff6-524dc352d104","remote_ip":"172.16.16.7","host":"172.16.16.38:3000","uri":"/forms/chromium/convert/url","method":"POST","path":"/forms/chromium/convert/url","referer":"","user_agent":"curl/8.5.0","status":403,"latency":805185461,"latency_human":"805.185461ms","bytes_in":208,"bytes_out":9}

curious, i've searched around for 'does not match the expression from the allowed list' and 403, but nothing is being returned. I see the error message in the code based, but I don't understand what the error means. Hoping someone will have some insight for me! 

Thank you!

**Comment by gulien** [comment]
Hey 👋 

Did you change the `--chromium-allow-list` flag value?

**Reply by cnewfeldt** [reply]
I did not, i will look in to this and report back. Thank you!

**Reply by cnewfeldt** [reply]
The value exist in my install of the URAID docker app. There is a 'Post Arguments' area with the following command 'gotenberg --chromium-disable-javascript=true --chromium-allow-list=file:///tmp/.*' 

Going to assume a permissions issue on my system of some kind. I will dig deeper. thank you again.

**Reply by gulien** [reply, marked as answer]
Remove the flag and you should be fine!

**Reply by cnewfeldt** [reply]
yup, that was it. Thank you!
